packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/handlers/resource-handlers.test.ts

```diff
@@ -1,6 +1,5 @@
-import { $TSContext, stateManager } from 'amplify-cli-core';
+import { $TSContext, stateManager, CognitoConfiguration } from 'amplify-cli-core';
```

Comment:
Taking another look through the code, `updatingAuth` is only used in `amplify-cli` and `amplify-category-auth`. I agree with you that it is best not to move this into core for that reason. I moved it back into `-auth`, and removed `updatingAuth` from the `$TSContext` exported in `-core`. I created a new union type in `-auth` that describes the `updatingAuth` property within the `auth` category. Since `amplify-cli` already depends on `-auth`, `-cli` can import the union type as necessary.

Reply:
Thank you.